symplyphysics/laws/electricity/transmission_lines/microstrip_lines/attenuation_coefficient_in_metall_of_microstrip_line_for_width_less_thickness.py

```diff
@@ -0,0 +1,74 @@
+from sympy import Eq, solve, pi, log
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    dimensionless,
+)
+
+## Description
+## The microstrip line is a dielectric substrate on which a metal strip is applied.
+## The effective width of a microstrip line is the width of such a flat capacitor, the electric intensity between the plates
+## of which is equal to the electric intensity in the dielectric of the substrate under the line strip.
+## The attenuation coefficient shows how many times the transmitted signal weakens per unit length of the microstrip line.
+
+## Law is: am = (1.38 * Rs / (h * Z0)) * ((32 - (Wef / h)^2) / (32 + (Wef / h)^2)) * F, where
+## F = 1 + (h / Wef) * (1 + 1.25 * t / (pi * W) + 1.25 * ln(4 * pi * W / t) / pi) (if h >= 2 * pi * Wef),
+## F = 1 + (h / Wef) * (1 - 1.25 * t / (pi * h) + 1.25 * ln(2 * h / t) / pi) (if h < 2 * pi * Wef),
```

Comment:
Separate to 2 laws.

Reply:
> Separate to 2 laws.

Then what are these two laws called?

Reply:
`... for_thickness_greater_than_effective_width_times_2pi`
`... for_thickness_greater_than_effective_width_less_than_effective_width_times_2pi`

I suggest to avoid such specific laws. We should stop narrowing down laws at some point and I think that this is somewhere near our limit.